rafay/utils.go

```diff
@@ -1492,10 +1493,48 @@ func expandVariableOptions(p []interface{}) *eaaspb.VariableOptions {
 		options.Override = expandVariableOverrideOptions(v)
 	}
 
+	options.DisplayMetadata = getExpandDisplayMetadata(opts)
+
 	return options
 
 }
 
+func getExpandDisplayMetadata(in map[string]interface{}) *structpb.Struct {
+	if v, ok := in["display_metadata"].(map[string]interface{}); ok && len(v) > 0 {
+		newMap := map[string]interface{}{}
+		for key, value := range v {
+
+			valueStr, ok := value.(string)
+			if !ok {
+				continue
+			}
+
+			if valueStr == "true" || valueStr == "false" {
```

Comment:
why do we need all of this ? In below section as well `strconv.Atoi(value.(string))` and setting the value in map.

What are the possible values we expect ?

Reply:
@nirav-rafay 
DisplayMetadata in backend is a RawJson field, but in terraform since we cannot have a dictionary with different value types, this code handles basic int, bool and string values.